identity-account/src/storage/memstore.rs

```diff
@@ -51,8 +53,9 @@ pub struct MemStore {
 impl MemStore {
   pub fn new() -> Self {
     Self {
-      expand: false,
+      expand: true,
```

Comment:
This defaults to printing private keys in debug strings, was changing this intentional? The `MemStore` is likely to be used only for testing/debugging during development and not in production, but defaulting to not potentially printing private keys in logs seems safer.